### ISSUE TITLE
local host access denied

### ISSUE THREAD

**Issue by lktoby** [Rigellute/spotify-tui#475]
hi, i tried to install spotify-tui on my newly installed arch linux laptop, but when i try to connect to the spotify api, it loaded a url, asked me to login to spotify, then said localhost:8888 has access denied. 
![image](https://user-images.githubusercontent.com/16471234/82752960-5b044480-9db1-11ea-823f-71dcc99d5b02.png)
i made sure my spotify app is open (installed via aur) and i have apache installed on my root system.

**Comment by Rigellute** [comment]
Hmm... is the `spt` process still running when you get this?

Perhaps your system is blocking the local web server that runs during this step?

**Comment by Rigellute** [comment]
Take a look at the comments in this issue for additional help https://github.com/Rigellute/spotify-tui/issues/476

**Comment by lktoby** [comment]
yes the process is still running when i get this, and im not sure if my system is blocking the local web server
